src/v/config/configuration.cc

```diff
@@ -1476,6 +1476,13 @@ configuration::configuration()
       "set in production.",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       false)
+  , disable_cluster_metadata_upload_loop(
+      *this,
+      "disable_cluster_metadata_upload_loop",
+      "Disables the cluster metadata upload loop.",
+      {.needs_restart = needs_restart::yes, .visibility = visibility::tunable},
```

Comment:
nit: Does this really need a restart? Could you thread the binding to the uploader and start/stop the uploader as the config changes?

Reply:
Good point, not really. I'll open an issue for this